swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+
+class Clothing(Item):
+    def __init__(self,category="Clothing",condition=0,age=0):
+        self.category=category
+        self.condition=condition
+        self.age=age
```

Comment:
There are two issues here, the first is that the category for `Clothing` must be the string "Clothing".   Making category an optional parameter here still makes it possible to pass in an alternative string for category, which could result in an instance of `Clothing` with a category that isn't "Clothing".   Using a string literal would ensure that the category is always "Clothing":
```suggestion
```suggestion
    def __init__(self, condition=0, age=0):
       self.condition=condition
       self.category="Clothing"
       self.age=age
```

Second, this is repeating work that is already being done in the Item constructor.  Using super().__init__(category, condition, age) would DRY the code here:
```suggestion
    def __init__(self, condition=0, age=0):
        super().__init__("Clothing", condition, age)
```